### DISCUSSION TITLE
Hello, is it possible to have a little help to help me configure the colorscheme material.nvim in LazyVim?

### DISCUSSION THREAD

**Discussion by maxlestage** [LazyVim/LazyVim#2494] [Q&A]
I open this little question, because I want to have a little push to configure material.nvim correctly in lazyvim.  
I created a dedicated file but the configuration seems to be going to be done very different. A little help or a boost would be welcome to understand once and not solicit your help unnecessarily next time.  

https://github.com/marko-cerovac/material.nvim 
<img width="1840" alt="Screenshot 2024-02-03 at 7 15 20 PM" src="https://github.com/LazyVim/LazyVim/assets/56438697/2c4bf648-7c5c-4fd3-acdf-4f8ba9b91723">

**Comment by maxlestage** [comment]
As you can see right now everything is purple, and it should be with Orange dominance : 

<img width="1840" alt="Screenshot 2024-02-03 at 7 17 40 PM" src="https://github.com/LazyVim/LazyVim/assets/56438697/dcbddd06-a6cd-481b-bca6-2348c12df11c">

**Comment by maxlestage** [comment]
I use it on VScode and they should be similar:  

<img width="1840" alt="Screenshot 2024-02-03 at 7 19 30 PM" src="https://github.com/LazyVim/LazyVim/assets/56438697/09149fd8-4b65-4156-8f04-6de374d193fa">

**Comment by dpetka2001** [comment]
It's a matter of the theme. I tried it in a `.cpp` file (because that is the example used in the README as well) and the colors matched as the ones in the repo.

I found a post in the official repo about official portings [here](https://github.com/material-theme/vsc-material-theme/discussions/1279). I tried the 2 Vim portings and I believe [this one](https://github.com/kaicataldo/material.vim) matches better with what you would expect compared to VsCode. There are still small differences.

My advice would be to use the one you linked on your opening post and change the highlight groups as suggested [here](https://github.com/marko-cerovac/material.nvim/blob/22a92aae1bba52cb8d34771e41e132ed0634bef5/README.md?plain=1#L192-L217) to have the colors that you want.
